src/hotspot/share/gc/g1/g1CodeCacheRemSet.cpp

```diff
@@ -40,104 +40,42 @@
 G1CodeRootSetTable* volatile G1CodeRootSetTable::_purge_list = NULL;
 
 size_t G1CodeRootSetTable::mem_size() {
```

Comment:
G1CodeRootSetTable::* definitions in `g1CodeCacheRemSet.cpp` make `g1CodeRootSetTable.hpp` seem redundant. Either add `g1CodeRootSetTable.cpp` or move the `G1CodeRootSetTable` declaration to `g1CodeCacheRemSet.hpp`

Reply:
I will merge them.